upload-api/tables/delegations.js

```diff
@@ -121,36 +121,35 @@ export function useDelegationsTable (dynamoDb, tableName, { bucket, invocationBu
       })
       const response = await dynamoDb.send(cmd)
 
-      const delegations = []
+      const delegationPromises = []
```

Comment:
Would be good to retain capturing the problematic CID in the error. Perhaps if this was:

```js
/** @type {Array<{ delegationCid, promise }>} */
const delegationPromises = []
```

You could use later to augment the error message appropriately.